test-cypress/integration/sparql-editor/lucene-connector.spec.js

```diff
@@ -0,0 +1,65 @@
+import {SparqlEditorSteps} from "../../steps/sparql-editor-steps";
+import {YasguiSteps} from "../../steps/yasgui/yasgui-steps";
+import {LuceneConnectorSteps} from "../../steps/lucene-connector-steps";
+import {ConnectorsStubs} from "../../stubs/yasgui/connectors-stubs";
+import {YasqeSteps} from "../../steps/yasgui/yasqe-steps";
+import {ErrorPluginSteps} from "../../steps/yasgui/plugin/error-plugin-steps";
+import {TablePluginSteps} from "../../steps/yasgui/table-plugin-steps";
+
+describe('Connectors - Lucene', () => {
+    let repositoryId;
+
+    beforeEach(() => {
+        repositoryId = 'sparql-editor-' + Date.now();
+        cy.intercept('GET', '/rest/monitor/query/count', {body: 0});
```

Comment:
Move this in a respective stub class

Reply:
This is like other test. I will move it in `QueryStubs` and will update other tests.

Reply:
done